common-practices-tools/contribution/contrib-first.md

```diff
@@ -0,0 +1,29 @@
+# Contrib First
```

Comment:
@swirtSJW I forgot that titles should have frontmatter as well, e.g. https://github.com/CivicActions/guidebook/blob/master/practice-areas/project-management/project-management-checklists.md?plain=1#L1-L3